### PR TITLE
Enabling the release-controller to operate on ReleasePayloads

### PR DESCRIPTION
**Pull request by bradmwilliams**

### CODE REVIEW

File: clusters/app.ci/trt/admin_rbac.yaml [hunk 1]
```diff
@@ -63,3 +63,16 @@ rules:
   verbs:
   - get
   - update
+- apiGroups:
+  - release.openshift.io
+  resources:
+  - releasepayloads
+  verbs:
+  - create
+  - delete
```
Comment (deads2k) [inline clusters/app.ci/trt/admin_rbac.yaml:72]: why should they have delete powers?

Comment (bradmwilliams) [reply clusters/app.ci/trt/admin_rbac.yaml]: Removed

File: clusters/app.ci/trt/admin_rbac.yaml [hunk 2]
```diff
@@ -63,3 +63,16 @@ rules:
   verbs:
   - get
   - update
+- apiGroups:
+  - release.openshift.io
+  resources:
+  - releasepayloads
+  verbs:
+  - create
```
[the remark this answers is not in the document]
Comment (bradmwilliams) [reply clusters/app.ci/trt/admin_rbac.yaml]: Removed

**Pull request merged**